crates/runtime/src/pallets.rs

```diff
@@ -75,6 +75,9 @@ impl pallet_starknet::Config for Runtime {
     type ProtocolVersion = ProtocolVersion;
     type ProgramHash = ProgramHash;
     type ExecutionConstants = ExecutionConstants;
+    type InvokeTransactionFilter = ();
+    type DeclareTransactionFilter = ();
+    type DeployAccountTransactionFilter = ();
```

Comment:
so we don't add the rules here by default?

Reply:
By default we want our Madara to be as extensive as possible: support all tx versions, all contract classes, etc.

Then other chains can use it to "do less", but we should still cover as much surface area as possible ourselves

Reply:
Got it, although, by default do we not want it to replicate Starknet Mainent (no V0 txs)

Reply:
Good question. I would say no on this specific point. But that's debatable.
In any case we made clear that Madara should not be used as it is and require a bit of customization